setup.py

```diff
@@ -1,6 +1,5 @@
 #!/usr/bin/env python
-
-import sys
+import sys  # lgtm [py/unused-import]
```

Comment:
Not sure I understand why we’re not just removing this? If it’s a similar type of hack to `multiprocessing` below we should add a comment.